shared/elements/gaia_sim_picker/script.js

```diff
@@ -10,6 +10,13 @@
  * Dependencies: gaia-menu web component.
```

Comment:
note: we use various others /shared scripts; for example /shared/js/option_menu.js has an error now.